src/step-input.scss

```diff
@@ -0,0 +1,153 @@
+@import "./settings";
+@import "./new-settings";
+@import "./mixins";
+@import "./functions";
+@import "./button";
+
+$block: #{$fd-namespace}-step-input;
+
+.#{$block} {
+  $fd-step-input-border-thin-width: 0.0625rem;
+  $fd-step-input-border-thick-width: 2 * $fd-step-input-border-thin-width;
+  $fd-input-group-button-active-focus-color: var(--sapContent_ContrastFocusColor);
+  $fd-button-icon-font-size: 0.875rem;
+  $fd-button-icon-font-size-cozy: 1rem;
+
+  @mixin iconSize($size) {
+    &::before,
+    &::after {
+      font-size: $size;
+    }
+  }
+
+  width: auto;
+  display: inline-block;
+  line-height: 0;
+
+  @include fd-fiori-focus() {
+    outline-color: $fd-input-group-button-active-focus-color;
+  }
+
+  @include fd-reset();
+  @include fd-rtl();
+
+  &__content-wrapper {
+    width: auto;
+    min-width: 2.75rem;
+    margin-top: 0.25rem;
+    margin-bottom: 0.25rem;
+    padding: 0;
+    box-sizing: border-box;
```

Comment:
`padding: 0` and `box-sizing: border-box` is already part of `fd-input-group` and can be removed here